torchci/lib/bot/webhookToDynamo.ts

```diff
@@ -103,9 +103,41 @@ async function handlePush(event: WebhookEvent<"push">) {
   });
 }
 
+async function handlePullRequestReview(
+  event: WebhookEvent<"pull_request_review">
+) {
+  const key_prefix = event.payload.repository.full_name + "/";
```

Comment:
No need for a `/` at the end here, since on L115 you are adding one between anyway. And yes, the existing code is broken in this respect :P

Reply:
I noticed that for PRs and issues, we use the number which would get overwritten on updates right? Do we not need previous records?

Reply:
yeah this is intentional, so that we have only the "latest" record, which simplifies queries.